root/static/scripts/common/MB.js

```diff
@@ -1,20 +1,9 @@
 /*
  * @flow
+ * This file is part of MusicBrainz, the open internet music database.
  * Copyright (C) 2009 Oliver Charles
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ * Licensed under the GPL version 2, or (at your option) any later version:
+ * http://www.gnu.org/licenses/gpl-2.0.txt
```

Comment:
This is not the newest blurb, see https://musicbrainz.org/doc/Development/Server_Coding_Style#Templates which occurs in half of JavaScript files already. Should be:

```
 * Copyright (C) 2009 Olivier Charles
 *
 * This file is part of MusicBrainz, the open internet music database,
 * and is licensed under the GPL version 2, or (at your option) any
 * later version: http://www.gnu.org/licenses/gpl-2.0.txt
```

Reply:
Dammit! :) I'll fix it today.